### PR TITLE
dart: cleanup code using recommended style

### PR DESCRIPTION
**Pull request by kevmoo**

### CODE REVIEW

**Comment by kevmoo** [comment]
@hanabi1224 – next time you're bored! 😄

File: bench/algorithm/http-server/2.dart [hunk 1]
```diff
@@ -6,30 +6,30 @@ import 'dart:math';
...
+  final n = arguments.isNotEmpty ? int.parse(arguments[0]) : 500;
   final port = 20000 + Random().nextInt(30000);
   // print(port);
 
   final serverReceivePort = ReceivePort();
   late final SendPort serverSendPort;
 
-  serverReceivePort.first.then((message) {
-    serverSendPort = message;
+  await serverReceivePort.first.then((message) {
```
Comment (hanabi1224) [inline bench/algorithm/http-server/2.dart:19]: Looks like it's causing deadlock here, below code should work
```dart
final task = serverReceivePort.first.then((message) {
    serverSendPort = message as SendPort;
    serverSendPort.send(port);
  });

  await Isolate.spawn(_startServer, serverReceivePort.sendPort);
  await task;
```

Comment (kevmoo) [reply bench/algorithm/http-server/2.dart]: Fixed! thanks!

**Comment by hanabi1224** [comment]
Thanks

**Pull request merged**